### PR TITLE
[MOD-85][Improvement] Move reset to didTransition

### PR DESCRIPTION
**Pull request by laurenbarker**
## Purpose
Move reset from `willTransition` to `didTransition`.

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -7,12 +7,26 @@ const SCROLL_RESET_DISABLED_ROUTES = ['preprints.provider.moderation', 'preprint
...
-        const disableResetScroll = SCROLL_RESET_DISABLED_ROUTES.includes(oldRoute) && SCROLL_RESET_DISABLED_ROUTES.includes(newRoute);
-        if (!disableResetScroll) {
+
+        this.set('transitioningFrom', Ember.Router._routePath(oldInfos));
+        this.set('transitioningTo', Ember.Router._routePath(newInfos));
+    },
+
+    didTransition() {
+        this._super(...arguments);
+        console.log(this.get('disableResetScroll'));
```
Comment (sheriefvt) [inline app/router.js]: there is console statement

Comment (laurenbarker) [reply app/router.js]: Ah, thanks!

File: app/router.js [hunk 2]
```diff
@@ -7,12 +7,25 @@ const SCROLL_RESET_DISABLED_ROUTES = ['preprints.provider.moderation', 'preprint
...
+            SCROLL_RESET_DISABLED_ROUTES.includes(this.get('transitioningTo'));
+    }),
+
     willTransition(oldInfos, newInfos) {
         this._super(...arguments);
-        const oldRoute = Ember.Router._routePath(oldInfos);
-        const newRoute = Ember.Router._routePath(newInfos);
-        const disableResetScroll = SCROLL_RESET_DISABLED_ROUTES.includes(oldRoute) && SCROLL_RESET_DISABLED_ROUTES.includes(newRoute);
-        if (!disableResetScroll) {
+
```
Comment (aaxelb) [inline app/router.js:21]: This is the minorest of minor things, but it bothers me that there's an empty line after `_super` here, but not in `didTransition`.

**Review by aaxelb — DISMISSED** [review]
Looks good! One tiny nitpick, which you can ignore and merge if you want.

File: app/router.js (hunk 2 above)
Comment (laurenbarker) [reply app/router.js:21]: Do you prefer newline or no newline?

Comment (aaxelb) [reply app/router.js:21]: No real preference besides consistency. I suppose I would probably put a newline?

Comment (laurenbarker) [reply app/router.js:21]: @aaxelb I added one because I like newlines.

Comment (aaxelb) [reply app/router.js:21]: Glad we're on the same page :1st_place_medal:

**Review by aaxelb — APPROVED** [review]

**Pull request merged**